modules/mono/csharp_script.h

```diff
@@ -103,7 +103,7 @@ class CSharpScript : public Script {
 	bool valid = false;
 	bool reload_invalidated = false;
 
-	bool builtin;
+	bool builtin = false;
```

Comment:
This actually seems unused, so it should be removed. No reference to it in the `.cpp`, and it's private.